.travis.yml

```diff
@@ -30,5 +11,4 @@ before_install:
       -Denvironment.application.name=${ENVIRONMENT_APP:-simpleapp}
       -Dags.version=${AGS_VERSION:-14.9}
       -Dacs.version=${ACS_VERSION:-7.1.0.1}"
-  -  if [ "$TRAVIS_JOB_NAME" = "build application" ]; then (cd alfresco-apa-java-rest-api/alfresco-apa-java-rest-api-lib; sh generate.sh); fi
-  -  if [ "$TRAVIS_JOB_NAME" = "build application" ]; then (cd alfresco-acs-java-rest-api/alfresco-acs-java-rest-api-lib; sh generate.sh); fi
+install: mvn $MAVEN_CLI_OPTS install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
```

Comment:
By default, the code will be generated every time the project is built. It would be great to have an option to skip the generation. E.g. when we are developing the event client. 

`mvn clean install -Dskip.generation`

Reply:
Yes, that's a very good idea. I'm going to bring back the `generate` maven profile, enable it by default and use the system variable `skip.generation` to disable it.